instance/models/database_server.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2016 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Instance app models - Database Server models
+"""
+
+# Imports #####################################################################
+
+from django.db import models
+
+from .utils import ValidateModelMixin
+
+# Models ######################################################################
+
+class DatabaseServer(ValidateModelMixin, models.Model):  # FIXME: Does this need to subclass ValidateModelMixin?
```

Comment:
yes, it does : )

Actually since everyone is tripping over this, it might be worth factoring it into a BaseModel object somewhere, which would then just need to be imported by all other models.